extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -56,6 +62,30 @@ def get(self, objectId: str = None):
                 application/json:
                   schema: Error
         """
+        single_user_group = (
+            DBSession().query(Group).filter(Group.name == self.current_user.username).first()
+        )
+        # grab streams accessible to user:
+        if single_user_group is not None:
+            streams = (
```

Comment:
I thought we were going to use `StreamUser` instead of associating streams with the single user groups?

Reply:
Fixed